.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+# Crate Pull Request Form
+
+## Description
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+`Your message here`
+## Type of change
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Other, Please explain:
```

Comment:
I really like this particular template! When I've used it before the only thing that came up is that sometimes for "Type of change" we also needed a "Testing" category, though not sure we will for this project.

Reply:
I remember them mentioning today that we will need to do testing, so I think that's an excellent suggestion. Yeah, the level of detail/coverage here is crate.

Reply:
I like using the checkboxes, but just as a heads up GH considers them part of a list so you will see something like `6/10` on PR requests

Reply:
Upvoting this for a testing option!

Reply:
I love adding Testing to the templates. These are our templates now. Dream Team!